data/remnant/remnant missions.txt

```diff
@@ -3466,6 +3466,45 @@ event "remnant: cognizance timer"
 
 
 
+mission "Remnant: Cognizance 19"
+	name "Deliver nav data"
+	description "Deliver navigational data on the new systems to <destination>."
+	landing
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+	to offer
```

Comment:
You may get the same planet as source and destination with this limited of a destination filter